.github/workflows/tempate-sync-pat.yml

```
@@ -0,0 +1,33 @@
+name: Template Sync with PAT
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)